src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -223,3 +223,101 @@ The span finishes once the operation has been executed. The span `status` is set
 When the operation throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and SentryEvent will be linked when viewing it on the **Issue Details** page in sentry.io.
 
 For more information see our [file I/O integration](/platforms/android/configuration/integrations/file-io/).
+
+### User Interaction Instrumentation
+
+The UI instrumentation, once enabled, captures transactions for a set of different user interactions, which include clicks, scrolls and swipes. The SDK composes the Transaction name out of the host Activity and the `id` of the view, which captured the user interaction; for example, `LoginActivity.login_button`. The Transaction operation is set to `ui.action` plus the interaction type (one of `click`, `scroll` or `swipe`).
+
+<Note>
+
+If the view does not have the `id` assigned, the transaction will not be captured, because it cannot be uniquely identified otherwise.
+
+</Note>
+
+_The UI instrumentation is disabled by default, but you may enable it by setting:_
+
+```xml {filename:AndroidManifest.xml}
+<manifest>
+    <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />
+</manifest>
+```
+
+The transaction finishes automatically after the specified [idleTimeout](/platforms/android/configuration/options/#idle-timeout) and all of its child spans are finished. The `idleTimeoout` defaults to 3 seconds; you can also disable the idle timeout by setting it to `null` - the transaction must be finished manually in this case.
```

Comment:
Numbers 1-9 have to be spelled out unless they appear in the code, so I've added what appears in the code followed by the spelled out description.

```suggestion
The transaction finishes automatically after it reaches the specified [idleTimeout](/platforms/android/configuration/options/#idle-timeout) and all of its child spans are finished. The `idleTimeoout` defaults to `3000` milliseconds (three seconds). You can also disable the idle timeout by setting it to `null`, but the transaction must be finished manually in this case.
```